sdks/go/pkg/beam/core/util/symtab/symtab.go

```diff
@@ -33,14 +32,13 @@ type SymbolTable struct {
 // New creates a new symbol table based on the debug info
 // read from the specified file.
 func New(filename string) (*SymbolTable, error) {
-	b, err := ioutil.ReadFile(filename)
+	f, err := os.Open(filename)
```

Comment:
defer f.Close() ?

Reply:
The debug libraries take a io.ReaderAt as their argument, taking ownership of that reader. So the file needs to stay open, so closing it here is incorrect.